pkg/api/postage.go

```diff
@@ -182,8 +182,15 @@ func (s *Service) postageGetStampsHandler(w http.ResponseWriter, r *http.Request
 	}
 
 	resp := postageStampsResponse{}
-	resp.Stamps = make([]postageStampResponse, 0, len(s.post.StampIssuers()))
-	for _, v := range s.post.StampIssuers() {
+	stampIssuers, err := s.post.StampIssuers()
+	if err != nil {
+		logger.Debug("get stamps: get issuers failed", "error", err)
```

Comment:
The `get stamps:` prefix is not needed because the logger already inserts it in the log message. The same is below.